CHANGELOG.md

```diff
@@ -80,7 +80,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (staking) [#18506](https://github.com/cosmos/cosmos-sdk/pull/18506) Detect the length of the ed25519 pubkey in CreateValidator to prevent panic.
 * (tx) [#18772](https://github.com/cosmos/cosmos-sdk/pull/18772) Remove misleading gas wanted from tx simulation failure log.
 * (tx) [#18852](https://github.com/cosmos/cosmos-sdk/pull/18852) Add `WithFromName` to tx factory.
-
+* (client) [#18929](https://github.com/cosmos/cosmos-sdk/pull/18929) Improve `client.Context.From` documentation. 
```

Comment:
No changelog needed for documentation changes.

Reply:
changelog removed

Reply:
I think something went wrong as I still see it

Reply:
strange, removed the changelog now.